test/modules/redfish_auth.py

```diff
@@ -31,21 +32,36 @@ def local_host_exception_setup():
         param = dumps({'username': 'admin', 'password': 'admin123', 'role': 'Administrator'})
         user_add_cmd = "curl -k -X POST -w '%{http_code}' -H 'Content-Type: application/json' -d '"
         user_add_cmd += param
-        user_add_cmd += "' https://127.0.0.1:" + USER_AUTH_PORT + "/api/2.0/users"
 
-        term = pxssh.pxssh()
-        term.SSH_OPTS = (term.SSH_OPTS
-                         + " -o 'StrictHostKeyChecking=no'"
-                         + " -o 'UserKnownHostsFile=/dev/null' ")
-        term.force_password = True
-        term.login(HOST_IP, SSH_USER, SSH_PASSWORD, port=SSH_PORT)
-        term.sendline(user_add_cmd)
-        index = term.expect(['201', '401', '403'], 10)
+        # add first user to remote rackhd directly
+        remote_user_add_cmd = user_add_cmd + \
+            "' https://{0}:{1}/api/2.0/users".format(HOST_IP, USER_AUTH_PORT)
```

Comment:
There's one tricky point here.
Normally the HOST_IP is localhost,
if not localhost, the 1st step will fail and the 2nd step ```ssh login && curl localhost``` will be executed.